test/kitchen/site-cookbooks/dd-agent-upgrade/recipes/default.rb

```diff
@@ -75,9 +75,16 @@
     package node['dd-agent-upgrade']['package_name'] do
       action :remove
     end
-    package node['dd-agent-upgrade']['package_name'] do
-      action :install
-      version node['dd-agent-upgrade']['version']
+    # package node['dd-agent-upgrade']['package_name'] do
```

Comment:
If we are keeping this as a comment, maybe add a TODO above with a link to the issue we are trying to fix?

Reply:
Good point, I'll add a comment there.